torch/distributions/exponential.py

```diff
@@ -24,6 +24,14 @@ class Exponential(Distribution):
     support = constraints.positive
     has_rsample = True
 
+    @property
+    def mean(self):
+        return self.rate.pow(-1)
```

Comment:
Is `.reciprocal()` any cheaper or more stable than `.pow(-1)`?

Reply:
I can confirm that for you. Yes, it is.